management/actions_test.go

```diff
@@ -238,6 +238,23 @@ func TestActionManager_Execution(t *testing.T) {
 	assert.Equal(t, http.StatusNotFound, err.(Error).Status())
 }
 
+func TestActionManager_LogSession(t *testing.T) {
+	setupHTTPRecordings(t)
+
+	expectedLogSession := &ActionLogSession{
+		Filters: []ActionLogSessionFilter{{
+			Key: auth0.String("action_id"),
+			Val: auth0.String("act_123"),
+		}},
+	}
+
+	err := m.Action.LogSession(expectedLogSession)
```

Comment:
Shouldnt we get back the log session here and compare it to `expectedLogSession`? Is that how this works?

Reply:
The `expectedLogSession` is where the result will be unmarshalled AND it also happens to be the payload. The convention took me some time to wrap my head around but it's used across the codebase.

Reply:
Yep indeed, the passed in object also gets additional parameters that might be returned from the endpoint marshalled directly into it.

Reply:
👍🏾